### PR TITLE
import-vm-apb template is using datavolumes not pvc

### PR DESCRIPTION
**Pull request by rawagner**
backport https://github.com/kubevirt/web-ui-components/pull/87 to v1.3

### CODE REVIEW

File: src/components/Wizard/CreateVmWizard/StorageTab.js [hunk 1]
```diff
@@ -425,7 +422,10 @@ class StorageTab extends React.Component {
       const values = {
```
Comment (suomiy) [inline src/components/Wizard/CreateVmWizard/StorageTab.js]: please could you refactor this so the if only mentions sourceType which are allowed?

**Comment by suomiy** [comment]
> backport #87 to v1.3
can you explain please a bit more why is this PR here twice?

File: src/utils/selectors.js [hunk 2]
```diff
@@ -4,12 +4,13 @@ export const getName = value => get(value, 'metadata.name');
 export const getNamespace = value => get(value, 'metadata.namespace');
 
 export const getGibStorageSize = (units, object) => {
-  const size = get(object, 'spec.resources.requests.storage');
+  const size = get(object, 'spec.resources.requests.storage') || get(object, 'spec.pvc.resources.requests.storage');
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/utils/selectors.js]: fixed

File: src/models/index.js [hunk 3]
```diff
@@ -17,15 +17,3 @@ export const ProcessedTemplatesModel = {
   apiGroup: 'template.openshift.io',
   namespaced: true
 };
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/models/index.js]: Can't say. But I would rather re-add it later if needed and keep the repo clean from stuff that is not used.

[the remark this answers is not in the document]
Comment (suomiy) [reply src/models/index.js]: ok

File: src/components/Wizard/CreateVmWizard/StorageTab.js (hunk 1 above)
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/StorageTab.js]: fixed

**Comment by mareklibra** [comment]
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1646947

**Pull request merged**